### ISSUE TITLE
[bug] - helm charts not published after migrating to gorelease

### ISSUE THREAD

**Issue by grzesuav** [metacontroller/metacontroller#621]
Missing on https://github.com/metacontroller/metacontroller/pkgs/container/metacontroller-helm

**Comment by grzesuav** [comment]
```
Run helm package /home/runner/work/metacontroller/metacontroller/deploy/helm/metacontroller/
  helm package /home/runner/work/metacontroller/metacontroller/deploy/helm/metacontroller/
  shell: /usr/bin/bash -e {0}
  env:
    HELM_EXPERIMENTAL_OCI: 1
Successfully packaged chart and saved it to: /home/runner/work/metacontroller/metacontroller/metacontroller-helm-v4.[3](https://github.com/metacontroller/metacontroller/runs/7947052203?check_suite_focus=true#step:5:3).7.tgz
0s
Run helm push metacontroller-helm-.*.tgz oci://ghcr.io/metacontroller
Error: metacontroller-helm-.*.tgz: no such file
Error: Process completed with exit code [1](https://github.com/metacontroller/metacontroller/runs/7947052203?check_suite_focus=true#step:6:1).
```

**Comment by grzesuav** [comment]
fixed in 4.3.7
